ast/Trees.h

```diff
@@ -456,9 +456,16 @@ class Send final : public Expression {
 public:
     core::NameRef fun;
 
-    static const int PRIVATE_OK = 1 << 0;
-    static const int REWRITER_SYNTHESIZED = 1 << 1;
-    u4 flags;
+    struct __attribute__((packed, aligned(1))) Flags {
```

Comment:
Do you need the packed and aligned attributes here?

Reply:
No you don’t. I thought I pushed a fix up to this. I’ll do it tomorrow